ZConfig/tests/test_datatypes.py

```diff
@@ -277,10 +276,16 @@ def check(value, family, address, self=self, convert=convert):
         if hasattr(socket, "AF_UNIX"):
             self.assertEqual(a1.family, socket.AF_UNIX)
             self.assertEqual(a2.family, socket.AF_UNIX)
-        else:
+        else: # pragma: no cover
             self.assertTrue(a1.family is None)
             self.assertTrue(a2.family is None)
 
+        convert = self.types.get('socket-binding-address')
+        check(":80",                 AF_INET, (defhost, 80))
+
+        convert = self.types.get('socket-connection-address')
+        check(":80",                 AF_INET, ("127.0.0.1", 80))
```

Comment:
I can only imagine I was thinking that looking like a table would make this more readable.  I wouldn't make the same mistake now.  (I certainly hope.)